renku/core/session/docker.py

```diff
@@ -160,13 +160,19 @@ def session_start(
                     ]
 
             # NOTE: set git user
+            image_data = self.docker_client().api.inspect_image(image_name)
+            working_dir = image_data.get("Config", {}).get("WorkingDir", None)
+
+            if working_dir is None:
+                working_dir = "/home/jovyan"
+
             environment = {}
-            volumes = [f"{str(client.path.resolve())}:/home/jovyan/work"]
+            volumes = [f"{str(client.path.resolve())}:{working_dir}/work"]
```

Comment:
better to use pathlib or similar for path concatenation? To avoid e.g. double forward slashes.